docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ssl-esp.xml

```diff
@@ -1,383 +1,647 @@
+        file to all nodes, Restart, and Certify</emphasis></title>
+
+        <para>Once your environment is set up as desired, you must copy the
+        configuration file out to the other nodes.</para>
+
+        <orderedlist numeration="arabic">
+          <listitem>
+            <para>If it is running, stop the system.</para>
+
+            <para>Make sure system is stopped before attempting to move the
+            environment.xml file.</para>
+          </listitem>
+
+          <listitem>
+            <para>Back up the original environment.xml file</para>
+
+            <programlisting># for example
+sudo cp /etc/HPCCSystems/environment.xml /etc/HPCCSystems/environment.bak
+</programlisting>
 
-        <para>Make sure system is stopped before attempting to move the
-        environment.xml file.</para>
-      </listitem>
+            <para>Note: the "live" environment.xml file is located in your
+            <emphasis role="bold">/etc/HPCCSystems/ </emphasis>directory.
+            ConfigManager works on files in <emphasis
+            role="bold">/etc/HPCCSystems/source </emphasis>directory. You must
+            copy the XML file from this location to make an environment.xml
+            file active.</para>
+          </listitem>
 
-      <listitem>
-        <para>Back up the original environment.xml file</para>
+          <listitem>
+            <para>Copy the NewEnvironment.xml file from the source directory
+            to the /etc/HPCCSystems and rename the file to
+            environment.xml</para>
 
-        <programlisting># for example
+            <programlisting># for example
+sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.xml
+</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Copy the <emphasis
+            role="bold">/etc/HPCCSystems/environment.xml</emphasis> to
+            the<emphasis role="bold"> /etc/HPCCSystems/ </emphasis>on every
+            node.</para>
+
+            <para>You might prefer to use a script to automate this step,
+            especially if you have many nodes. See the Example Scripts section
+            in the Appendix of the Installing and Running the HPCCPlatform
+            manual.</para>
+          </listitem>
+
+          <listitem>
+            <para>Restart the HPCC system and certify the components as
+            usual.</para>
+          </listitem>
+        </orderedlist>
+      </sect2>
+    </sect1>
+
+    <sect1 id="ssl4Roxie">
+      <title>Configuring SSL for Roxie</title>
+
+      <para>Just as HPCC has Secure Sockets Layer (SSL) protocol for ESP there
+      is similar secure socket layer for Roxie. (You may have already
+      completed these steps if you configured SSL for ESP as described in the
+      previous section) Please refer to <link linkend="ssl4esp">SSL For
+      ESP</link> section for more information on setting up SSL key and
+      certificate pairs.</para>
+
```

Comment:
I do not think 100 (or even 500) nodes all reading a small file over nfs at the same time would cause any significant delays.  No one here uses nfs, but I have seen it used elsewhere.